src/DurableTask.SqlServer.AzureFunctions/DurableTask.SqlServer.AzureFunctions.csproj

```diff
@@ -4,7 +4,11 @@
   <Import Project="../common.props" />
   
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <DefineConstants>$(DefineConstants);FUNCTIONS_V4</DefineConstants>
```

Comment:
I think bundles still targets TFM netcoreapp3.1.
But I know the host targets net6.0

I hope that still means that at runtime the FUNCTIONS_v4 variable will be defined. Not super sure about the resolution rules here. But wanted to call this out.

Reply:
V4 bundles were changed to target net6.0 a few months ago, so we should be good to also target net6.0: https://github.com/Azure/azure-functions-extension-bundles/pull/407.